test/2d.jl

```diff
@@ -119,6 +119,11 @@ using Rotations, StaticArrays, Base.Test
         show(io, MIME("text/plain"), r)
         str = String(take!(io))
         @test startswith(str, "2×2 RotMatrix{Float64}:")
```

Comment:
@tkoolen I'm getting test errors here, and output more like `"2×2 RotMatrix{2,Float64,4}:"`. Can you reproduce?

Reply:
Oops, I thought I had tested that properly on 0.7. I think I actually prefer the current output (since `RotMatrix{Float64}` is not actually a type), so I'll change the test. But let me know if you disagree.